src/Api/Api.php

```diff
@@ -100,6 +100,10 @@ public function getDatamodel(string $datamodelName): ?Datamodel
             return null;
         }
 
+        if (empty($response)) {
```

Comment:
Lepsie by bolo kontrolovat HTTP kod response a vyhodit exception (v `clientGetRequest`) ... v pripade chyby moze body obsahovat data, ale nechces ich parsovat. ... ale ked to ide, tak OK.

Reply:
Teda v pripade, ze ti to nevrati prazdnu response s kodom `200` ... co snad nie.

Reply:
no problém je v tom že http kód je pořád 200 jen s prázdnými daty (to je teď novinka :))

Reply:
aha :) tak to je z ich strany dost blba novinka, ... potom ok :+1:

Reply:
souhlasím :(